lib/pf/provisioner/intune.pm

```diff
@@ -294,7 +293,12 @@ sub verify_compliance {
                 if ($entry->{complianceState} eq 'compliant') {
                     $logger->info("Device $mac is compliant.");
                     return $TRUE;
-                } else {
+                } 
+                elsif ($entry->{complianceState} eq 'inGracePeriod') {
+					$logger->info("Device $mac is InGracePeriod.");
```

Comment:
incorrect indentation

Reply:
Hi Fabrice, sorry I missed in the developer guide about tabs.  Converted to spaces.